src/axom/inlet/docs/sphinx/advanced_types.rst

```diff
@@ -155,8 +198,8 @@ In Lua, the following operations on the ``Vector`` type are supported (for ``Vec
 #. Dimension retrieval: ``d = u.dim``
 #. Component retrieval: ``d = u.x``, ``d = u.y``, ``d = u.z``
 
-Retrieving Functions from an Input File
----------------------------------------
+Accessing
+---------
 
 To retrieve a function, both the implicit conversion and ``get<T>`` syntax is supported.  For example,
 a function can be retrieved as follows:
```

Comment:
The rest of this section likely needs to be updated. It seems to be at a different exposition level than the previous part of this section. E.g. what is a `tag`?
(perhaps this is clarified by the snippet that did not get generated)?